.travis.yml

```diff
@@ -33,6 +33,7 @@ script:
 after_success:
   - yarn test:codacy-upload
   - yarn coverage
+  - yarn a11y:demo
```

Comment:
is it demo we need to test?

Reply:
Yes, Demo app is referncing the ccd-case-ui-toolkit components